test/tests/userobjects/interface_user_object/interface_mp_real_user_object_QP.i

```diff
@@ -0,0 +1,194 @@
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 2
+    xmax = 2
+    ny = 2
+    ymax = 2
+  []
+  [./subdomain1]
+    input = gen
+    type = SubdomainBoundingBoxGenerator
+    bottom_left = '0 0 0'
+    top_right = '1 1 0'
+    block_id = 1
+  [../]
+  [./master0_interface]
+    type = SideSetsBetweenSubdomainsGenerator
+    input = subdomain1
+    master_block = '0'
+    paired_block = '1'
+    new_boundary = 'master0_interface'
+  [../]
+  [./break_boundary]
+    input = master0_interface
+    type = BreakBoundaryOnSubdomainGenerator
+  [../]
+[]
+
+[Variables]
+  [./u]
+    order = FIRST
+    family = LAGRANGE
+    block = 0
+  [../]
+
+  [./v]
+    order = FIRST
+    family = LAGRANGE
+    block = 1
+  [../]
+[]
+
+[Kernels]
+  [./diff_u]
+    type = CoeffParamDiffusion
+    variable = u
+    D = 2
+    block = 0
+  [../]
+  [./diff_v]
+    type = CoeffParamDiffusion
+    variable = v
+    D = 4
+    block = 1
+  [../]
+  [./source_u]
+    type = BodyForce
+    variable = u
+    function = 0.1*t
+  [../]
+[]
+
+[InterfaceKernels]
+  [./master0_interface]
+    type = PenaltyInterfaceDiffusionDot
+    variable = u
+    neighbor_var = v
+    boundary = master0_interface
+    penalty = 1e6
+  [../]
+[]
+
+[BCs]
+  [./u]
+    type = VacuumBC
+    variable = u
+    boundary = 'left_to_0 bottom_to_0 right top'
+  [../]
+  [./v]
+    type = VacuumBC
+    variable = v
+    boundary = 'left_to_1 bottom_to_1'
+  [../]
+[]
+
+[Preconditioning]
+  [./SMP]
+    type = SMP
+    full = TRUE
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  solve_type = 'NEWTON'
+
+  petsc_options_iname = '-pc_type -pc_factor_mat_solver_package'
+  petsc_options_value = ' lu       superlu_dist                '
```

Comment:
Are you doing LU in order to control the iteration count?

Reply:
no control on the number of linear and nonlinear iterations. The test doesn't care as long as it can go forward without warring out

Reply:
K, if you end up doing another commit, go ahead and remove these lines then. It's best if new users who come along and look at our test files as examples don't default to using LU

Reply:
done